alg/teca_damper.cxx

```diff
@@ -0,0 +1,217 @@
+#include "teca_damper.h"
+
+#include "teca_mesh.h"
+#include "teca_array_collection.h"
+#include "teca_variant_array.h"
+#include "teca_metadata.h"
+#include "teca_cartesian_mesh.h"
+
+#include <algorithm>
+#include <iostream>
+#include <deque>
+#include <set>
+#include <cmath>
+#include <complex.h>
+
+using std::deque;
+using std::vector;
+using std::set;
+using std::cerr;
+using std::endl;
+
+//#define TECA_DEBUG
+namespace {
```

Comment:
in the code style used by teca, braces go on the following line. eg
```C++
for (int i = 0; i < n; ++i)
{
    printf("%d\n",i);
}
```
Could you please fix those?

Reply:
done!